src/AfricasTalking.php

```diff
@@ -24,17 +23,11 @@ class AfricasTalking
 
 	public function __construct($username, $apiKey)
 	{
-		if($username === 'sandbox') {
-			$this->baseDomain = self::BASE_SANDBOX_DOMAIN;
-		} else {
-			$this->baseDomain = self::BASE_DOMAIN;
-		}
-
-		$this->baseUrl = "https://api." . $this->baseDomain . "/version1/";
-		$this->voiceUrl = "https://voice." . $this->baseDomain . "/";
-		$this->paymentsUrl = "https://payments." . $this->baseDomain . "/";
-		$this->contentUrl = ($username === "sandbox") ? ($this->baseUrl) : ("https://content." . $this->baseDomain . "/version1/");
-		$this->checkoutTokenUrl = "https://api." . $this->baseDomain . "/";
+		$this->baseUrl = "https://api." . $this->base_domain() . "/version1/";
+		$this->voiceUrl = "https://voice." . $this->base_domain(). "/";
+		$this->paymentsUrl = "https://payments." . $this->base_domain() . "/";
+		$this->contentUrl = ($username === "sandbox") ? ($this->baseUrl) : ("https://content." . $this->base_domain() . "/version1/");
+		$this->checkoutTokenUrl = "https://api." . $this->base_domain() . "/";
 
 		if ($username === 'sandbox') {
```

Comment:
Given the tenary check on line 29, we actually don't need this if block. I think this was missed from a previous commit.